examples/dishwasher-app/silabs/data_model/dishwasher-thread-app.matter

```diff
@@ -2516,96 +2500,24 @@ endpoint 0 {
   }
 
   server cluster SoftwareDiagnostics {
-    emits event SoftwareFault;
-    callback attribute threadMetrics;
-    callback attribute currentHeapFree;
-    callback attribute currentHeapUsed;
-    callback attribute currentHeapHighWatermark;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
     callback attribute featureMap;
     callback attribute clusterRevision;
-
-    handle command ResetWatermarks;
-  }
-
-  server cluster ThreadNetworkDiagnostics {
```

Comment:
This is another thread app removing the thread diagnostics cluster, is there a reason why we want to do this?